debug/src/check-props.js

```diff
@@ -0,0 +1,28 @@
+
+if (process.env.NODE_ENV !== 'production') {
+  var ReactPropTypesSecret = 'SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED',
+      loggedTypeFailures = {};
+}
+
+export function checkPropTypes(typeSpecs, values, location, componentName, getStack) {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Same as above, this check can be omitted here :+1:

Reply:
in 9bcfb14 as well